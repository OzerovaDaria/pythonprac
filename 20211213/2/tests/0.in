[0, 1, 23, 23, 33, 0, 0, 2, 1, 1, 22, 23, 23, 24, 25, 0]
