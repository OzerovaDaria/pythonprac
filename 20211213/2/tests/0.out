[0, 0, 0, 0, 1, 1, 1, 2, 22, 23, 23, 23, 23, 24, 25, 33]
